### ISSUE TITLE
Icon downloads on every route change

### ISSUE THREAD

**Issue by dmbaranov** [ayroblu/ssr-create-react-app-v2#20]
First of all, let me say thanks for this awesome work! You implemented a great idea of using CRA without ejecting it together with SSR, really like it! (If there would be a HMR I would like it even more! Because now I have to re-build client part every time I change it). 

But one thing I noticed is that icon is downloading again and again each time you change the route. Not sure what causes this behaviour, but probably you do. It's not a big problem though, but would be cool to fix it somehow. Thanks!

<img width="1280" alt="screen shot 2017-09-15 at 01 33 13" src="https://user-images.githubusercontent.com/10092277/30458830-013c14a2-99b6-11e7-876e-f9405b8bcfcc.png">

**Comment by ayroblu** [comment]
Without looking at the source code, I'm guessing the favicon doesn't exist so it's sending the raw HTML which isn't cached and no 404 right now

**Comment by dmbaranov** [comment]
Well, actually this screenshot was taken from the freshly installed code of this repository :)

**Comment by ayroblu** [comment]
Yeah that's what I'm saying, my code probably doesn't have it, feel free to put your own favicon in there and try again
